cache/cache.go

```diff
@@ -68,13 +69,13 @@ func RebuildCmd(c Cache, src, dst string) (err error) {
 	src = filepath.Clean(src)
 	src, err = filepath.Abs(src)
 	if err != nil {
-		return err
+		return fmt.Errorf("%s, absolute path failed for %s", err, src)
```

Comment:
@bradrydzewski are you ok with this format? Within dronme we are using strings like `fmt.Errorf("Failed to get absolute path for %s. %s", src, err)`